homeassistant/components/simplisafe/__init__.py

```diff
@@ -303,29 +286,58 @@ def v3_only(coro):
 class SimpliSafe:
     """Define a SimpliSafe API object."""
 
-    def __init__(self, hass, api, systems, config_entry):
+    def __init__(self, hass, api, config_entry):
         """Initialize."""
         self._api = api
         self._config_entry = config_entry
         self._emergency_refresh_token_used = False
         self._hass = hass
         self.last_event_data = {}
-        self.systems = systems
+        self.systems = None
 
-    async def _update_system(self, system):
-        """Update a system."""
-        try:
+    async def async_init(self):
+        """Initialize the data class."""
+        self.systems = await self._api.get_systems()
+
+        # Register the base station for each system:
+        for system in self.systems.values():
+            self._hass.async_create_task(
```

Comment:
Why is it important to create tasks for this?

Reply:
I figured this would allow the event loop to execute these at its leisure, as opposed to `await`ing them during this init process. Feedback welcome!

Reply:
Ok.